src/uvw/util.hpp

```diff
@@ -11,6 +11,12 @@
 #include <array>
 #include <uv.h>
 
+#ifdef _WIN32
```

Comment:
Interesting. From the logs it seems that you are not using revision 14 of the standard (actually, it mentions the fact that with C++14 `const` won't be implied anymore), or at least you have not all the available features at your disposal. Anyway, good catch. Thank you for the help.